pkg/epp/scheduling/framework/plugins.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 const (
-	ProfilePickerType          = "ProfilePicker"
+	ProfileHandlerType         = "ProfileHandler"
```

Comment:
this should be reverted. it’s the extension point name and is used to store metrics for time to perform the extension point.